.bumpversion.toml

```diff
@@ -1,24 +1,23 @@
 [tool.bumpversion]
 current_version = "1.0.3"
 tag_prefix = "v"
-tag = true
```

Comment:
i removed this, because otherwise, it will create the tag locally every time I test it.

in the workflow, the tag is explicitly added, and I've updated the dev readme to include this as well now.